src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -961,6 +968,7 @@ private void resolveToken(ref CORINFO_RESOLVED_TOKEN pResolvedToken)
                 pResolvedToken.hClass = ObjectToHandle(method.OwningType);
 
 #if READYTORUN
+            _compilation.NodeFactory.ThrowIfTypeRequiresRuntimeJit(method.OwningType);
```

Comment:
You should not need these "catch all" checks anymore. For example, `typeof(Vector<int>)`  can be compiled just fine even in R2R.

Reply:
OK, thanks for pointing that out, fixed in 8th commit.